### PR TITLE
SISRP-19014 - GL5.4 - My Academics page - Class Enrollment card - BerkeleyTime feature

### PR DESCRIPTION
**Pull request by dfkaye**
https://jira.berkeley.edu/browse/SISRP-19014

add Berkeleytime link to schedule of classes

### CODE REVIEW

**Comment by dfkaye** [comment]
One green travis, one red (alas)

File: src/assets/templates/widgets/enrollment/explore.html [hunk 1]
```diff
@@ -51,6 +51,12 @@
         <a href="https://calstudentstore.berkeley.edu/textbook">Class Textbooks: ASUC Bookstore</a>
       </li>
     </ul>
+    <div>
+      <h4>ASUC Sponsored Planning Site</h4>
```
Comment (johncrossman) [inline src/assets/templates/widgets/enrollment/explore.html]: Just curious, @dfkaye  Why have this in an `<h4>` separated out from the `<ul>`?

Comment (dfkaye) [inline src/assets/templates/widgets/enrollment/explore.html]: @johncrossman It's a way to match the visual design which has that heading above the single link with its own list bullet.  see design jira at https://jira.berkeley.edu/browse/SISRP-15091

**Comment by johncrossman** [comment]
Thanks @dfkaye  Looks good.

**Pull request merged**